wiki/People/The_Team/id.md

```diff
@@ -11,47 +11,47 @@ tags:
+| ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) | Manajer Project Loved, penasihat bidang pengembangan komunitas, koordinator program Featured Artist |
 | ![][flag_ES] [Trosk-](https://osu.ppy.sh/users/3469385) | Pengasuh umum, penyelenggara turnamen |
 | ![][flag_AU] [Zallius](https://osu.ppy.sh/users/55) | Spesies yang terancam punah |
 
-Di samping nama-nama di atas, osu! juga memiliki [tim dukungan akun](Account_support_team) (*account support team*) yang bergerak secara terpisah untuk membantu menangani berbagai permasalahan yang bersifat pribadi.
+Di samping nama-nama di atas, osu! juga memiliki [tim dukungan akun](Account_support_team) (*account support team*) yang bergerak secara terpisah untuk menangani berbagai permasalahan yang bersifat pribadi.
 
-## Gugus pengguna
+## Kelompok pengguna
 
-Berikut ini adalah gugus-gugus pengguna (*user groups*) yang tersusun atas anggota-anggota komunitas terpilih beserta dengan tanggung jawab dan kontribusinya masing-masing terhadap ekosistem osu!. Pada umumnya, anggota gugus-gugus pengguna di bawah ini dapat dikenali dari warna khusus pada forum, warna khusus pada kanal obrolan, serta gelar dan/atau lencana khusus yang disematkan kepada masing-masing anggota.
+osu! memiliki berbagai kelompok pengguna (*user group*) yang tersusun atas para anggota komunitas terpilih dengan tanggung jawab dan fungsinya masing-masing. Pada umumnya, anggota kelompok pengguna tertentu dapat dikenali dari warna khusus pada forum dan kanal obrolan serta gelar dan/atau lencana khusus yang mereka miliki.
 
 | Nama | Deskripsi |
 | :-- | :-- |
-| [Beatmap Nominator](Beatmap_Nominators) | Menjaga standar kualitas beatmap dan mendorong beatmap-beatmap yang dianggap layak ke dalam kategori Qualified |
-| [Developer](Developers) | Membuat osu! senantiasa tampil menawan melalui implementasi berbagai fitur canggih dan penumpasan berbagai macam *bug* |
+| [Beatmap Nominator](Beatmap_Nominators) | Menjaga standar kualitas beatmap dan mendorong beatmap yang dianggap layak ke dalam kategori Qualified |
+| [Developer](Developers) | Membuat osu! tampil menawan dengan mewujudkan berbagai fitur baru dan memastikan osu! bebas dari *bug* |
 | [Global Moderation Team](Global_Moderation_Team) | Mengawasi forum dan kanal obrolan |
 | [Nomination Assessment Team](Nomination_Assessment_Team) | Mengevaluasi kinerja Beatmap Nominator |
 | [osu! Alumni](osu!_Alumni) | Para pendahulu yang telah banyak berjasa pada masanya |
-| [Project Loved Team](Project_Loved_Team) | Mengukuhkan beatmap-beatmap yang dicintai oleh komunitas |
-| [Support Team](Support_Team) | Menjawab berbagai pertanyaan yang terlontar pada sub-forum Help dan Feature Request |
+| [Project Loved Team](Project_Loved_Team) | Mengukuhkan beatmap yang dicintai oleh komunitas |
+| [Support Team](Support_Team) | Melayani berbagai pertanyaan dan permintaan yang diajukan pada sub-forum Help dan Feature Request |
```

Comment:
> Help / Feature Request assistance
it seems they are more like assisting in help and feature sub-forum request only. still unsure about answering each question given.

Reply:
yup, changed this into `Memberikan bantuan pada sub-forum Help dan Feature Request` to reflect the words in the English article better